src/library/Book.java

```diff
@@ -47,21 +47,11 @@ public String toString() {
                 '}';
     }
 
-    @Override
-    public int compareTo(Book o) {
-        return this.getTitle().toLowerCase().compareTo(o.getTitle().toLowerCase());
-    }
-
     @Override
     public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         Book book = (Book) o;
         return id == book.id && Objects.equals(title, book.title) && Objects.equals(genre, book.genre);
     }
-
-    @Override
-    public int hashCode() {
```

Comment:
nonono. hashCode and equals functions should come together. revert this rows please

Reply:
Done